content/md/first-steps/manage-your-rules.md

```diff
@@ -38,7 +38,8 @@ The number of affected products will appear on the right side of your screen in
 # Execute rules
 
 :::info
-Rules are regularly executed after a bulk action or after imports with rules.
+Rules are automatically executed according to your cron configuration. By default, they are executed daily at 5 am UTC. 
```

Comment:
```suggestion
Rules are automatically executed according to your cron configuration. By default, they are executed daily at 5 am UTC.  
```
Two spaces are needed at the end of this line to force going to the next line. It's Markdown specifications. See here for example: https://www.markdownguide.org/basic-syntax/#line-breaks